### ISSUE TITLE
Feature Request: Redesign to match modern plex clients

### ISSUE THREAD

**Issue by JonzaUK** [pannal/plex-for-kodi#143]
Feature Request for a redesign to match modern plex clients UX design. feels like PM4K is showing its age in the design department, might just be my opinion but I think a design matching the look of modern would look amazing.

**Comment by pannal** [comment]
Yeah. That's in the back of my mind but very far in the future. Being mostly the only active maintainer, this is a huge endeavor.

Maybe in some future version, or if someone else tackles it.

**Comment by firasdib** [comment]
Can you add some documentation on how to get started? There is nothing in the readme.

**Comment by zenhuw** [comment]
I want to start with OSD UI , will try working on it in the next few months

**Comment by pannal** [comment]
> I want to start with OSD UI , will try working on it in the next few months

Haha, that's probably the "most modern" part of the PM4K UI :D
